### ISSUE TITLE
checkLocalURL() fails with URL format 'http://[computer_name].local

### ISSUE THREAD

**Issue by jakemoves** [jakemoves/cohort-server#187]
When running locally, it's super helpful to be able to access the admin site from a mobile device connected to the same network at i.e. `http://jakemoves.local:3000/admin-v2`, where 'jakemoves' is the computer name. This runs into trouble with your current code.

My suggestion:
- get `window.location.hostname`
- split on `:`, chuck the port if present
- split on `.`
- check the returned array:
  - if the first element is `localhost` OR the second element is `local`...
- construct the serverURL from `window.location.protocol + '//' + window.location.host + '/api/v2'` and continue as you do

**Comment by helloCaller** [comment]
Right now we have have the initial state of serverURL getting constructed from window.location.protocol + '//' + window.location.host + '/api/v2' , that addresses this issue. Will the change from /admin-v2 to /admin mean we'll also cut the v2 off the end of this?

**Comment by jakemoves** [comment]
No, the versioning on the API is not related. It just took a long time to ‘retire’ the hacky admin site I built last winter.
